spec/policies/cart_item_policy_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
```

Comment:
after removing unnecessary policies from Cart CartItemds and other Classes you must clear many tests.